addons/website/static/src/js/content/menu.js

```diff
@@ -660,9 +660,20 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
                 return;
             }
         }
+        // Get the previously focused element of the page.
+        const focusedEl = this.el.ownerDocument.querySelector(":focus");
```

Comment:
If the focus is on the admin header, it will be lost.
```suggestion
        const focusedEl = this.el.ownerDocument.querySelector(":focus") || window.frameElement.ownerDocument.querySelector(":focus");
```

Reply:
`window.frameElement` does not exist if this header is not present (after a refresh for example) :thinking: Do you prefer checking its existence before ( `window.frameElement && ...`) or simply ignoring this issue ?